themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
See http://tiddlywiki.com/#Highlight%20Plugin and http://tiddlywiki.com/plugins/tiddlywiki/highlight/#%24%3A%2Fplugins%2Ftiddlywiki%2Fhighlight%2Fhighlight.css